restapi/src/main/java/io/stargate/web/resources/IndexResource.java

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
We don't need to add new functionality to v1

Reply:
Oh, nice! 😄 Gonna remove that asap.